e2e-identity/tests/utils/fmk.rs

```diff
@@ -753,7 +753,8 @@ impl E2eTest {
         self.display_token("OAuth Access token", access_token, None, &dex_pk);
 
         if let Some(refresh_token) = oauth_token_response.refresh_token() {
-            self.display_token("OAuth Refresh token", refresh_token.secret(), None, &dex_pk);
+            // We don't call display_token here with the refresh token because Keycloak
```

Comment:
Even if don't verify the refresh token, shouldn't we still write to the test log?

Reply:
Hm, good point. We should probably write it, but instead of writing "Invalid signature" write something like "Unable to verify signature of type <X>".